mobile/src/common/enums/colors/background-color.enum.ts

```diff
@@ -0,0 +1,8 @@
+enum BackgroundColor {
```

Comment:
Remove colors folder and all unnecessary enums

Reply:
sorry, did't add all changes to git